scripts/Interface/reduction_gui/widgets/reflectometer/launch_peak_back_selection_1d.py

```diff
@@ -216,14 +216,14 @@ def create_main_frame(self):
         self.canvas = FigureCanvas(self.fig)
         self.canvas.setParent(self.main_frame)
 
-#        self.axes = self.fig.add_subplot(111)
+        #        self.axes = self.fig.add_subplot(111)
```

Comment:
I would remove the space between the hash and code. So removing the hash would allow the code to run